projects/igniteui-angular/src/lib/tabbar/tabbar.component.ts

```diff
@@ -130,10 +163,19 @@ export class IgxBottomNavComponent implements AfterViewInit {
     public get itemStyle(): string {
         return this._itemStyle;
     }
+
+    /**
+     *@hidden
+     */
+    public get hasContentTabs(): boolean {
+        return (this.contentTabs && this.contentTabs.toArray().length > 0);
```

Comment:
contentTabs should have 'length' property, no need to cast to array.

Reply:
Addressed.